api-tests/.env.local

```diff
@@ -7,3 +7,6 @@ CORE_BACK_PUBLIC_ENCRYPTION_KEY='{"kty": "RSA","n": "0465qJwo8nCkC2tvV4niuWF6IM6
 ORCHESTRATOR_REDIRECT_URL="http://localhost:4500/callback"
 JAR_SIGNING_KEY='{"kty":"EC","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}' # pragma: allowlist secret
 LOCAL_AUDIT_EVENTS=true
+ASYNC_QUEUE_DELAY=2
+
+# Also requires ASYNC_QUEUE_NAME to be specified in .env or as an environment variable
```

Comment:
(q for my own learning) does this mean that if we're missing a variable in this .env.local file, it'll look for that variable in the .env file?